src/styles/blocks/star.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background: url(../images/star-active.svg) center no-repeat;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
```

Comment:
What if rating is 3 stars ? or 2? Here you handle only the case when rating is 4 stars.